### ISSUE TITLE
a11y/media-prefers-reduced-motion doesn't detect override for rule with multiple selectors

### ISSUE THREAD

**Issue by ghost** [YozhikM/stylelint-a11y#57]
These rules trigger an error with `a11y/media-prefers-reduced-motion`:

```css
.Zoom-img,
.Zoom-wrap {
  position: relative;
  transition: all var(--Zoom-transition);
  z-index: 1;
}

@media screen and (prefers-reduced-motion) {
  .Zoom-img,
  .Zoom-wrap {
    transition: none;
  }
}
```

Modifying the above to rules with single selectors gets rid of the error:

```css
.Zoom-wrap {
  position: relative;
  transition: all var(--Zoom-transition);
  z-index: 1;
}

@media screen and (prefers-reduced-motion) {
  .Zoom-wrap {
    transition: none;
  }
}
```

So it seems that this rule does not parse css rules with multiple selectors correctly. Neither of the above cases should return a linting error.
